sunkit_pyvista/plotter.py

```diff
@@ -97,7 +97,7 @@ def set_view_angle(self, angle: u.deg):
             The viewing angle.
         """
         view_angle = angle.to_value(u.deg)
-        if not view_angle > 0 and view_angle <= 180:
+        if not (view_angle > 0 and view_angle <= 180):
```

Comment:
Can you add a test to check this is working again?

Reply:
Sure